.travis.yml

```diff
@@ -1,27 +1,34 @@
 ---
-dist: xenial
 language: python
 matrix:
   include:
     - python: 2.6
       dist: trusty
+      # If the role contains python code that should be linted and tested, say
+      # this by setting LSR_USE_PYTHON environment variable to the value that
+      # evaluates as true in the role's repository settings page at
+      # https://travis-ci.org/
+      if: env(LSR_USE_PYTHON) IS NOT blank
```

Comment:
Is it possible to configure `LSR_USE_PYTHON` in the `.travis/config.sh` file? I would prefer to have as much configuration as possible in one place. Splitting the config between the repo and the web interface makes it harder to check/understand.

Reply:
That's a good idea. It makes a sense as `./travis/config.sh` contains project specific CI settings.

Reply:
Oh, I forgot what was the main reason behind having `LSR_USE_PYTHON` set at the Travis side: saving the Travis resources. I am not sure if when `LSR_USE_PYTHON` will be set in `.travis/config.sh` then it will become accessible via `env()` from `.travis.yml`. Maybe there is a trick how to include additional configuration files into `.travis.yml` but I didn't find it yet.

Reply:
After some considerations, having a configuration in a one place took priority over saving Travis resources when role does not contain parts written in Python[*], so `LSR_USE_PYTHON` was omitted completely.

[*] Sadly, it seems that there is no other way how to save Travis resources other than the trick with `if: env(VAR) IS NOT blank`. Note that other alternatives of `if: ...` can result in changing the common CI configuration frequently, like in case if we use repo name tests hardcoded in `.travis.yml`, which is not the path we want to follow.